staging/src/k8s.io/client-go/kubernetes/fake/clientset_generated_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fake
+
+import (
+	"context"
+	v1 "k8s.io/api/core/v1"
+	policy "k8s.io/api/policy/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func TestNewSimpleClientset(t *testing.T) {
+	client := NewSimpleClientset()
+	client.CoreV1().Pods("default").Create(context.Background(), &v1.Pod{
+		ObjectMeta: meta_v1.ObjectMeta{
+			Name:      "pod-1",
+			Namespace: "default",
+		},
+	}, meta_v1.CreateOptions{})
+	client.CoreV1().Pods("default").Create(context.Background(), &v1.Pod{
+		ObjectMeta: meta_v1.ObjectMeta{
+			Name:      "pod-2",
+			Namespace: "default",
+		},
+	}, meta_v1.CreateOptions{})
+	_ = client.CoreV1().Pods("default").EvictV1(context.Background(), &policy.Eviction{
```

Comment:
add an assertion that the return type was correct

Reply:
received, assertion has been added